src/poggit/module/build/RepoListBuildPage.php

```diff
@@ -104,9 +104,9 @@ protected function displayRepos(array $repos) {
             $opened = "false";
             if(count($repo->projects) === 1) $opened = "true";
             ?>
-            <div class="toggle" data-name="<?= $repo->full_name ?> (<?= count($repo->projects) ?>)"
+            <div class="repotoggle" data-name="<?= $repo->full_name ?> (<?= count($repo->projects) ?>)"
                  data-opened="<?= $opened ?>">
-                <h2><a href="<?= $home ?>ci/<?= $repo->full_name ?>">Projects</a> in
+                <h2><a href="<?= $home ?>ci/<?= $repo->full_name ?>"></a>
```

Comment:
I suppose we can still add a link to the RepoBuildPage for that repo? Just like the GitHub logo link.